src/ServiceControl/ServiceControl.csproj

```diff
@@ -34,7 +34,7 @@
     <PackageReference Include="NServiceBus.NLog" Version="3.0.0" />
     <PackageReference Include="NServiceBus.Raw" Version="3.2.2" />
     <PackageReference Include="Microsoft.Owin.Hosting" Version="4.1.1" />
-    <PackageReference Include="ServiceControl.Contracts" Version="2.0.0" />
+    <PackageReference Include="ServiceControl.Contracts" Version="3.0.0-PullRequest0046.9" />
```

Comment:
Same here, incorrect package.

Reply:
It is the new contracts, it will be updated once we release it